Sources/Build/YAML.swift

```diff
@@ -15,6 +15,14 @@ protocol YAMLRepresentable {
 extension String: YAMLRepresentable {
     var YAML: String {
         if self == "" { return "\"\"" }
+        for c in utf8 {
+            switch c {
+            case "@".utf8.first!, " ".utf8.first!, "-".utf8.first!, "&".utf8.first!:
```

Comment:
Yes, I think so -- was shooting for a NIFC ("no intended functionality change") performance fix with this, but I do think this is missing cases... I might end up reworking this for performance reasons anyway, I'll try and audit this then.